src/app/server/Mdns.h

```diff
@@ -19,26 +19,37 @@
 
 #include <core/CHIPError.h>
 #include <stddef.h>
+#include <support/BitFlags.h>
 
 namespace chip {
 namespace app {
 namespace Mdns {
 
+enum class CommissioningMode
+{
+    kDisabled,
+    kEnabled,
+    kEnabledAsAdditionalCommissioning
+};
+
 /// Start operational advertising
 CHIP_ERROR AdvertiseOperational();
 
 /// Set MDNS commissioner advertisement
 CHIP_ERROR AdvertiseCommissioner();
 
 /// Set MDNS commissionable node advertisement
-CHIP_ERROR AdvertiseCommissionableNode();
+CHIP_ERROR AdvertiseCommissionableNode(CommissioningMode mode);
 
-/// Set MDNS advertisement
-// CHIP_ERROR Advertise(chip::Mdns::CommssionAdvertiseMode mode);
-CHIP_ERROR Advertise(bool commissionableNode);
+/// (Re-)starts the minmdns server using default state
+// Commissioning mode is enabled if device has not yet been commissioned
```

Comment:
Please use `///` to continue the doc comment for consistency. I am unsure if doxygen keeps following // and generally using 3 everywhere seems to make things align somewhat nicer.

Reply:
fixed